auto/package.json

```diff
@@ -2,7 +2,8 @@
     "name": "chart.js-auto",
     "private": true,
     "description": "auto registering package",
```

Comment:
do we still need this file? If so, it'd be nice to explain in the comment here why it's necessary

Reply:
@benmccann Do you mean package.json or auto.js ?

Reply:
@benmccann For example, webpack support `exports` field only since v5. So for v4 we should keep package.json

Reply:
alright. can we update the description here to explain that?

Reply:
@benmccann How?)

Reply:
We could change it from `"description": "auto registering package",` to something like `"description": "auto registering package. this file exists to support bundlers without exports support such as webpack 4",` (same in `helpers/package.json`)

Reply:
@benmccann done 👍